File: Plus/Serializer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Newtonsoft.Json.Linq;

namespace Plus
{
    public class Serializer
    {
        private string _json;

        public string json
        {
            get { return _json; }
            private set { _json = value; }
        }

        public Serializer(string json)
        {
            this.json = json;
        }

        public List<string> keys()
        {
            return JObject.Parse(json).Properties().Select(p => p.Name).ToList();
        }

        public string snippet(string key)
        {
            string value = (string)JObject.Parse(json)[key];
            return value;
        }

    }
}

File: SnippetsPlus/Serializer.vb
Imports Newtonsoft.Json.Linq

Public Class Serializer
    Private _json As String

    Public Property json As String
        Get
            Return _json
        End Get
        Private Set(ByVal value As String)
            _json = value
        End Set
    End Property

    Public Sub New(ByVal json As String)
        Me.json = json
    End Sub

    Public Function keys() As List(Of String)
        Return JObject.Parse(json).Properties().[Select](Function(p) p.Name).ToList()
    End Function

    Public Function snippet(ByVal key As String) As String
        Dim value As String = CStr(JObject.Parse(json)(key))
        Return value
    End Function
End Class
